### PR TITLE
feat(Auto LiC): Known limitations

### PR DESCRIPTION
**Pull request by barbnewrelic**
Added per SMEs request.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
Board request failed for unknown reason. Closing and reopening this PR.

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx [hunk 1]
```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
...
-
 If your APM agent doesn't support our automatic logs in context solution yet, you can continue to use our [manual logs in context solutions](/docs/logs/logs-context/logs-in-context), and forward your logs via our infrastructure agent or supported third-party forwarder.
 
+## Known limitations [#limitations]
+
+APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+
+Here are some additional known limitations:
+
+* Startup logs or host information are not available. This is because logs are not collected until after the app is loaded.
```
Comment (kford-newrelic) [inline src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:80]: Should this just reflect "startup logs are not available", with no mention of host information? Are we saying that host information is available after startup?

As the reason why, should this be changed to "...until after the _agent_ is loaded", to be more specific/accurate?

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx [hunk 2]
```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
...
+* If you are using Kubernetes, the logs will not get decoration from the Kubernetes API. This is because the logs never touch the host or exist in a place where the API can be called.
```
Comment (kford-newrelic) [inline src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:81]: I wonder if this might be confusing, since a customer could look at their file system and would see logs, etc. Maybe we could just remove this and not explain? If we need to explain though, then we'd have to explain that we decorate via instrumentation, separate and apart from the writing of the logs out to the filesystem.

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx [hunk 3]
```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
...
+* When an APM event is triggered, you may not see all stack traces in the associated logs in context for Java or .NET agents. As a workaround, you can change your drop filter rules.
```
Comment (kford-newrelic) [inline src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:82]: Is this APM event related? Or should this read "When an exception is thrown in your application..."

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx [hunk 4]
```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
...
+
+For example, Fluentd can add the `processID` from the entity that generated the log, but APM logs can't see that. In Kubernetes, the API is called to add metadata, but that can't be seen from within the application. In these situations, we recommend that you use automatic logs in context but do not ship the logs from the application. Instead, continue to use Fluentd, Fluent Bit, or another solution to forward the log files.
```
Comment (kford-newrelic) [inline src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:84]: The location/indentation on this paragraph confused me a bit; is this example specific to the last bullet mentioned or something else? It wasn't clear to me what it was related to.

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx (hunk 1 above)
Comment (barbnewrelic) [reply src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:80]: Sorry, my notes about this were garbled. I'll fix. Thanks for setting me straight!

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx (hunk 2 above)
Comment (barbnewrelic) [reply src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:81]: @kford-newrelic  Typically we document what the system does, not what it doesn't do. But in this particular situation I was requested to add this info. I'll take another stab at rewriting this and have my original SME validate the wording. Thanks again for your thoughtful comments!

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx (hunk 4 above)
Comment (barbnewrelic) [reply src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:84]: @kford-newrelic  Good point. I'll double check.

File: src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx [hunk 5]
```diff
@@ -69,12 +69,19 @@ Not every language or logging framework is supported yet. The following are our
...
 
+## Known limitations [#limitations]
+
+APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+
+Here are some additional known limitations:
+
+* Startup logs are not available until after the agent is loaded.
+* If you are using Kubernetes, be aware that we decorate the logs via instrumentation, not from the Kubernetes API. This is separate and apart from the writing logs out to the filesystem. The logs never touch the host or exist in a place where the API can be called.
+* When an exception is thrown from your application, you may not see all stack traces in the associated logs in context for Java or .NET agents. As a workaround, you can change your drop filter rules.
```
Comment (meiao) [inline src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:82]: In Java, no stack trace will ever be sent when an exception is logged. While the language here covers this case, it may give the impression that some might be sent.

Comment (barbnewrelic) [reply src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx:82]: Thanks for making that clarification!

**Comment by barbnewrelic** [comment]
Approved by SMEs.
